contracts/authorization/src/tests/processor.rs

```diff
@@ -1046,7 +1044,12 @@ fn queue_shifting_when_not_retriable() {
                                 message_type: MessageType::CosmwasmExecuteMsg,
                                 message: Message {
                                     name: "will_error".to_string(),
-                                    params_restrictions: None,
+                                    params_restrictions: Some(vec![
+                                        ParamRestriction::MustBeIncluded(vec![
+                                            "will_error".to_string(),
+                                            "error".to_string(),
+                                        ]),
+                                    ]),
```

Comment:
Can you explain? added restrictions but not added tests?

Reply:
the test is done right after this. I'm sending this message right after this so I confirm that the restriction is working (exactly this must be sent)